ui/src/index.js

```diff
@@ -1,17 +1,18 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import "./index.css";
-import App from "./App";
-import reportWebVitals from "./reportWebVitals";
-import { ChakraProvider } from "@chakra-ui/react";
+import React from 'react';
```

Comment:
not related to this file - but not sure why package-lock.json has changes when package.json didn't have any new packages? best to leave it out of this commit if no changes

Reply:
it might be me. because there was some errors with chakra module and i deleted node_modules and package-lock.json and reinstall everything. either that or installed the ignite font